setup.cfg

```diff
@@ -25,7 +25,8 @@ disable = all
 # W0611 Unused import %s
 # R1710 Either all return statements in a function should return an expression,
 #       or none of them should.
-enable = C0326, C1801, F0401, E0602, E0603, W0601, W0611, R1710
+# R0401 Cyclic import %s
+enable = C0326, C1801, F0401, E0602, E0603, W0601, W0611, R1710, R0401
```

Comment:
can you use this as the new pylint section?
```
[pylint]
#[MESSAGES CONTROL]
msg-template = {path}:{symbol}:{line:3d},{column}: {msg}
enable = all
disable =
    abstract-method,
    arguments-differ,
    attribute-defined-outside-init,
    bad-continuation,
    broad-except,
    consider-iterating-dictionary,
    consider-merging-isinstance,
    consider-using-enumerate,
    dangerous-default-value,
    deprecated-method,
    duplicate-code,
    fixme,
    global-statement,
    invalid-name,
    invalid-unary-operand-type,
    keyword-arg-before-vararg,
    line-too-long,
    locally-disabled,
    method-hidden,
    missing-docstring,
    no-else-return,
    no-member,
    no-name-in-module,
    no-self-use,
    pointless-string-statement,
    protected-access,
    raising-format-tuple,
    redefined-argument-from-local,
    redefined-builtin,
    redefined-outer-name,
    reimported,
    superfluous-parens,
    super-init-not-called,
    suppressed-message,
    too-few-public-methods,
    too-many-arguments,
    too-many-branches,
    too-many-instance-attributes,
    too-many-lines,
    too-many-locals,
    too-many-public-methods,
    too-many-statements,
    undefined-loop-variable,
    unidiomatic-typecheck,
    unnecessary-lambda,
    unnecessary-pass,
    unused-argument,
    unused-format-string-argument,
    unused-variable,
    unused-wildcard-import,
    useless-super-delegation,
    useless-suppression,
    wildcard-import,
```

Reply:
Done.